modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/util/ConfigurationModelToDDMFormValuesConverter.java

```diff
@@ -201,6 +203,31 @@ else if (type.equals(DDMFormFieldType.SELECT)) {
 		ddmFormFieldValue.setValue(localizedValue);
 	}
 
+	private void _validateDDMFormValuesWithOverrideConfiguration(
+		DDMFormValues ddmFormValues) {
+
+		LocalizedValue localizedValue = new LocalizedValue();
```

Comment:
Is it working to declare this variable outside the loop?
Because I see that `localizedValue` (Map) will get "bigger" because of `localizedValue.addString()`. This method is adding (`put()`) more values into this Map. Then, this Map is being added to each `ddmFormFieldValue` with different values.